lib/src/web_socket/mock/w_socket.dart

```diff
@@ -16,113 +16,14 @@ import 'dart:async';
 
 import 'package:w_transport/src/mocks/web_socket.dart'
     show MockWebSocketInternal;
-import 'package:w_transport/src/web_socket/common/w_socket.dart';
-import 'package:w_transport/src/web_socket/w_socket.dart';
+import 'package:w_transport/src/web_socket/mock/web_socket.dart';
+import 'package:w_transport/src/web_socket/web_socket.dart';
 
-abstract class MockWSocket implements WSocket {
-  static Future<WSocket> connect(Uri uri,
+abstract class MockWSocket extends MockWebSocket implements WebSocket {
```

Comment:
Yes, but for a different reason. In #214 I'm going to add a separate test util class for controlling the behavior of the mock web socket server, which will obviate the need for this `MockWSocket` class to be exported publicly. Then in #207 I will deprecate `MockWSocket` along with other unnecessary mock classes.